framework/auth/core.py

```diff
@@ -373,6 +375,30 @@ def create_confirmed(cls, username, password, fullname):
         user.date_confirmed = user.date_registered
         return user
 
+    def get_cookie(self, secret):
```

Comment:
I see the point. `settings` are really global to the whole app (useful in both framework and website), so it's ok to import it here.